### ISSUE TITLE
can't find the cron job for scraping

### ISSUE THREAD

**Issue by jedahan** [atduskgreg/ofxaddons.com#127]
```bash
~/c/ofxaddons.com ❯❯❯ bundle exec rake cron --trace                            ⏎ master ⬆ ◼
rake aborted!
Don't know how to build task 'cron'
/Users/jedahan/.homebrew/Cellar/ruby/2.2.3/lib/ruby/2.2.0/rake/task_manager.rb:62:in `[]'
/Users/jedahan/.homebrew/Cellar/ruby/2.2.3/lib/ruby/2.2.0/rake/application.rb:149:in `invoke_task'
/Users/jedahan/.homebrew/Cellar/ruby/2.2.3/lib/ruby/2.2.0/rake/application.rb:106:in `block (2 levels) in top_level'
/Users/jedahan/.homebrew/Cellar/ruby/2.2.3/lib/ruby/2.2.0/rake/application.rb:106:in `each'
/Users/jedahan/.homebrew/Cellar/ruby/2.2.3/lib/ruby/2.2.0/rake/application.rb:106:in `block in top_level'
/Users/jedahan/.homebrew/Cellar/ruby/2.2.3/lib/ruby/2.2.0/rake/application.rb:115:in `run_with_threads'
/Users/jedahan/.homebrew/Cellar/ruby/2.2.3/lib/ruby/2.2.0/rake/application.rb:100:in `top_level'
/Users/jedahan/.homebrew/Cellar/ruby/2.2.3/lib/ruby/2.2.0/rake/application.rb:78:in `block in run'
/Users/jedahan/.homebrew/Cellar/ruby/2.2.3/lib/ruby/2.2.0/rake/application.rb:176:in `standard_exception_handling'
/Users/jedahan/.homebrew/Cellar/ruby/2.2.3/lib/ruby/2.2.0/rake/application.rb:75:in `run'
/Users/jedahan/.homebrew/bin/rake:33:in `<main>'
```

**Comment by jedahan** [comment]
Looksl like it got deleted in 42a698a, specifically https://github.com/atduskgreg/ofxaddons.com/commit/42a698a50f7df4ef1b5206f67616202806ead750#diff-52c976fc38ed2b4e3b1192f8a8e24cffL6

**Comment by jamezilla** [comment]
Sorry, it changed to a script/runner task a while back:
`rails r 'Importer.new.run(no_cache: true)'`

Importer.rb lives in /lib

Fixed in 4e44afa
